# Django will run "git log" to determine its current version. Not only is this
# a bit slow, but will this return a meaningless value as we are not using
# Django's actual git repository.

diff --git a/django/utils/version.py b/contrib/django/utils/version.py
index e0a8286..7edca84 100644
--- a/django/utils/version.py
+++ b/django/utils/version.py
@@ -22,7 +22,7 @@ def get_version(version=None):
 
     sub = ''
     if version[3] == 'alpha' and version[4] == 0:
-        git_changeset = get_git_changeset()
+        git_changeset = '+local'
         if git_changeset:
             sub = '.dev%s' % git_changeset
 
